swap/swap_test.go

```diff
@@ -1795,9 +1795,81 @@ func TestSwapLogToFile(t *testing.T) {
 		t.Fatal(err)
 	}
 	logString := string(b)
+
 	if !strings.Contains(logString, "sending cheque") {
 		t.Fatalf("expected the log to contain \"sending cheque\"")
 	}
+
+}
+
+func TestSwapActions(t *testing.T) {
+	// create a log dir
+	logDirDebitor, err := ioutil.TempDir("", "swap_test_log")
+	log.Debug("creating swap log dir")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(logDirDebitor)
+
+	// set the log dir to the params
+	params := newDefaultParams(t)
+	params.LogPath = logDirDebitor
+
+	testBackend := newTestBackend()
+	defer testBackend.Close()
+	// create test swap
+	testSwap, storeTestSwap := newBaseTestSwapWithParams(t, beneficiaryKey, params, testBackend)
```

Comment:
you aren't doing anything with `storeTestSwap` except removing it later.

i can't see a need for calling `newBaseTestSwapWithParams`, use `newTestSwap` instead, and simplify this test

Reply:
Fixed --> Simplified with `newTestSwap`